src/WebJobs.Extensions.NotificationHub/Config/Converter.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Azure.NotificationHubs;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.NotificationHubs
+{
+    internal class Converter
+    {
+        public void AddConverters(ref IConverterManager converterManager)
+        {
```

Comment:
This may look nicer if you made it an extension method on IConverterManager? and called it AddNotificationHubConverters()?

Then you could call converterManager.AddNotificationHubConverters() from your config...